### DISCUSSION TITLE
How to setup MQTT to connect Home Assistant?

### DISCUSSION THREAD

**Discussion by HercegAndras** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#524] [Q&A] [answered]
Checked the documents.
I put user/pwd and the IP address of the Home Assistant (2023.2.5) to the MQTT config of layz spa controller. 
Then at HA side I tried to add a HA MQTT device with the IP of the layz spa controller.
but cannot connect.

I am using HA in docker, but currently don't have any MQTT devices. (MQTT newbie here)
What I am missing?
Thanks!

UPDATE:
Okay, at the mean time I've just realized, that because I'm not using the HAOS, I do not have the add-ons (mosquito broker for MQTT) option. So either I need to install separately in docker, or I need the full HA OS to have an MQTT broker.

**Comment by visualapproach** [comment, marked as answer]
Yes you need a broker with a user name for the layzspa

**Comment by HercegAndras** [comment]
yepp, thanks!
